### PR TITLE
bug fix: can't read too many direntrys

### PR DESCRIPTION
**Pull request by caizejun**
### PR types
 Bug fixes 

### PR changes
Others

### Describe
bug fix: can't read too many direntrys

### CODE REVIEW

**Review by luoyuedong — APPROVED** [review]

File: pkg/fs/client/meta/meta_kv.go [hunk 1]
```diff
@@ -1329,19 +1380,32 @@ func (m *kvMeta) Readdir(ctx *Context, inode Ino, entries *[]*Entry) syscall.Err
...
 		}
-		insertDirEntry := &entryItem{
-			ino:    inode,
-			mode:   dirInodeItem.attr.Mode,
-			expire: now.Add(m.entryTimeOut).Unix(),
-			done:   entryDone,
-		}
+		return nil
+	})
+	if err != nil {
```
Comment (tornado404) [inline pkg/fs/client/meta/meta_kv.go:1393]: 打印一个日志，上面的代码段太长，出错了好排查

**Review by tornado404 — APPROVED** [review]
ok

File: pkg/fs/client/meta/meta_kv.go (hunk 1 above)
Comment (caizejun) [reply pkg/fs/client/meta/meta_kv.go:1393]: ok

Comment (caizejun) [reply pkg/fs/client/meta/meta_kv.go:1393]: ok

**Pull request merged**